docs/use-cases/tokenization/index.page.tsx

```diff
@@ -0,0 +1,395 @@
+  },
+  {
+    id: "ntf-tokens",
+    title: "Non-transferable Tokens",
+    description: (
+      <>
+        {translate(
+          "Native support for nontransferable items such as identity tokens, airline credits, and consumer rewards, honored by all on-chain participants."
+        )}
+      </>
+    ),
+  },
+  {
+    id: "royalties",
+    title: "Royalties",
+    subtitle: "Secures Creator Compensation:",
+    description: (
+      <>
+        {translate(
+          "Reliably collect your percentage of the sale price of your tokens."
+        )}
+      </>
+    ),
+  },
+];
+
+const stats = [
+  {
+    id: "nfts-minted",
+    stat: "20K",
+    description: "NFTs minted",
+  },
+  {
+    id: "nft-trading-vol",
+    stat: "~1000",
+    description: "NFT trading volume",
+  },
+  {
+    id: "network-speed",
+    stat: "0.4",
+    description: "Network speed (sec)",
+  },
+  {
+    id: "fee-per-tx",
+    stat: "$0.001",
+    description: "Avg fee per NFT tx",
+  },
+];
+
+const projects = [
+  {
+    id: "xrpcafe",
+    label: "XRP cafe",
+    url: "https://xrp.cafe",
+  },
+  {
+    id: "onXRP",
+    label: "onXRP",
+    url: "https://nft.onxrp.com",
+  },
+  {
+    id: "xaman",
+    label: "Xaman labs",
+    url: "https://github.com/XRPL-Labs/Xaman-App",
+  },
+  {
+    id: "amy",
+    label: "amy.network",
+    url: "https://token.amy.network",
+  },
+  {
+    id: "sologenic",
+    label: "Sologenic",
+    url: "https://sologenic.org/trade",
+  },
+  {
+    id: "carbonland",
+    label: "Carbonland trust",
+    url: "https://www.carbonlandtrust.com",
+  },
+  {
+    id: "nautilus",
+    label: "Nautilus wallet",
+    url: "https://github.com/nautls/nautilus-wallet",
+  },
+  {
+    id: "evernode",
+    label: "Evenode",
+    url: "https://evernode.org",
+  },
+  {
+    id: "raised-in-space",
+    label: "Raised in space",
+    url: "https://raisedinspace.com",
+  },
+];
+
+const articles = [
+  {
+    time: "JAN 2024",
+    title: "XRP Price Prediction: Will Ripple Make Waves in 2024?",
+    url: "https://www.forbes.com/advisor/au/investing/cryptocurrency/xrp-price-prediction",
+  },
+  {
+    time: "JAN 2024",
+    title: "XRP Could Surge Massively in 2024",
+    url: "https://u.today/xrp-could-surge-massively-at-beginning-of-2024-heres-why",
+  },
+  {
+    time: "NOV 2023",
+    title: "NFTs and the XRPL: A Marriage of Art and Technology",
+    url: "https://medium.com/@MariaSolorzano/title-nfts-and-the-xrpl-a-marriage-of-art-and-technology-cf76a0432693",
+  },
+];
+
+const FeaturedProjects = () => {
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  const handlePrev = () => {
+    if (currentIndex > 0) {
+      setCurrentIndex(currentIndex - 1);
+    }
+  };
+
+  const handleNext = () => {
+    if (currentIndex < projects.length - 3) {
+      setCurrentIndex(currentIndex + 1);
+    }
+  };
+
+  const ProjectCard = ({ project, index }) => (
+    <a
+      className={`col float-up-on-hover ${index % 2 === 0 ? "even" : "odd"}`}
```

Comment:
To make the cards use light mode colors, add the `card` class here (requires some other SCSS changes but also makes some other SCSS unnecessary)

```suggestion
      className={`col card float-up-on-hover ${index % 2 === 0 ? "even" : "odd"}`}
```